models/.gitattributes

```diff
@@ -4,4 +4,5 @@ dfp-models/* filter=lfs diff=lfs merge=lfs -text
 abp-models/* filter=lfs diff=lfs merge=lfs -text
 log-parsing-models/* filter=lfs diff=lfs merge=lfs -text
 root-cause-models/* filter=lfs diff=lfs merge=lfs -text
-datasets/** filter=lfs diff=lfs merge=lfs -text
+datasets/training-data/** filter=lfs diff=lfs merge=lfs -text
+datasets/validation-data/** filter=lfs diff=lfs merge=lfs -text
```

Comment:
You should keep this all sorted if you can.

Reply:
sorted alphabetically